js/components/colorSelector.js

```diff
@@ -11,20 +10,24 @@ var ColorSelector = React.createClass({
     });
   },
 
+  handleColorSelection: function(color) {
+    this.props.onColorChange(color);
+  },
+
   render: function() {
     var colorItems = [];
     for (var i = 0; i < this.props.colors.length; i++) {
       colorItems.push(
-        <div className={this.setClassname(this.props.colors[i])}>
-          <div className={"oo-color-item oo-color-item-" + this.props.colors[i]} onClick={this.props.onColorChange.bind(this, this.props.colors[i])} key={i}></div>
+        <div className={this.setClassname(this.props.colors[i])} key={i}>
+          <a className={"oo-color-item oo-color-item-" + this.props.colors[i]} onClick={this.handleColorSelection.bind(this, this.props.colors[i])}></a>
         </div>
```

Comment:
* resolve bind error from react
  * onClick event handlers need to be on clickable elements (ie - link, button) to work on mobile browsers